docs/applications/build-your-own/02-azure-app-service.md

```diff
@@ -0,0 +1,288 @@
+---
+id: applications-build-your-own-azure
+slug: /applications/build-your-own/azure-app-service
+title: Azure App Service
+sidebar_label: Azure App Service
+---
+# Table of contents
```

Comment:
Overal, I really like the organization of this. And how you link to other parts of our docs 👌